src/Types/UnbalancedTransaction.purs

```diff
@@ -64,7 +84,7 @@ newtype UnbalancedTx = UnbalancedTx
   { transaction :: Transaction
   , requiredSignatories :: Map PaymentPubKeyHash (Maybe PaymentPubKey)
   , utxoIndex :: Map TxOutputRef ScriptOutput
-  , validityTimeRange :: POSIXTimeRange
+  , validityTimeRange :: POSIXTimeRange -- Should we remove this? I think we have this info inside Transaction?
```

Comment:
We probably can remove this. We should also do something about `requiredSignatories` and `required_signers` which I believe is similarly duplicated